views/home.hbs

```diff
@@ -1,28 +1,28 @@
-  <div class="container">
-    <div class="row row-header">
-      <div class="col-sm-8 homeheader text-center">
-        <h1>Die Lernwerkstatt-Berlin/Treptow</h1>
-        <p>
-          Die Lernwerkstatt ist ein altersübergreifender Lernort für Lernende
-          in, außerhalb oder neben der Schule.
-        </p>
-        <img src="https://res.cloudinary.com/hrscywv4p/image/upload/c_fill,fl_lossy,h_200,w_200,f_auto,g_faces:auto,q_auto/v1/253787/6b3c1ba6-333a-4dfb-80b1-686bd4b9e0db_s4dlam.png"
-          class="img-fluid rounded-circle" alt="Lernwerkstatt Logo" />
-      </div>
-      <div class="col-sm-4 calendar-equal">
-        <div class="card bg-primary text-white card-calendar">
-          <div class="card-header">
-            <h2 class="text-center">Kalender</h2>
-          </div>
-          <ul class="list-group list-group-flush">
-            {{#each calendar}}
-            {{> home/event this}}
-            {{/each}}
-          </ul>
+<div class="container">
+  <div class="row row-header mt-5">
+    <div class="col-sm-8 homeheader text-center">
+      <h1>Die Lernwerkstatt-Berlin/Treptow</h1>
+      <p>
+        Die Lernwerkstatt ist ein altersübergreifender Lernort für Lernende
+        in, außerhalb oder neben der Schule.
+      </p>
+      <img src="https://res.cloudinary.com/hrscywv4p/image/upload/c_fill,fl_lossy,h_200,w_200,f_auto,g_faces:auto,q_auto/v1/253787/6b3c1ba6-333a-4dfb-80b1-686bd4b9e0db_s4dlam.png"
```

Comment:
it'll be better if we store images on our server, instead of relying on external links, they might broke.